api/tests/unit/cat/assessment_test.js

```diff
@@ -327,6 +327,22 @@ describe('Unit | Model | Assessment', function() {
       // then
       expect(assessment.filteredChallenges).to.deep.equal([keep1, keep2, keep3]);
     });
+
+    it('should not filtered by priority skill with maxDifficulty less than 3 when there is no answers', function() {
```

Comment:
CQU : retirer `with maxDifficulty less than 3`
contexte => `when there is no answers`